test/acceptance/tests/basic/terraform/basic-install/main.tf

```
@@ -49,7 +49,7 @@ variable "launch_type" {
 variable "consul_ecs_image" {
   description = "Consul ECS image to use."
   type        = string
-  default     = "public.ecr.aws/hashicorp/consul-ecs:0.4.1"
+  default     = "docker.mirror.hashicorp.services/hashicorpdev/consul-ecs:32c1d86"
```

I think this should be using the `0.4.1-dev` tag.
```suggestion
  default     = "docker.mirror.hashicorp.services/hashicorpdev/consul-ecs:0.4.1-dev"
```